docs/_documentations/container-registry-credentials.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: docs
+title: Container registry credentials for Codewind
+description: Specifying container registries for Codewind
+keywords: docker, containers, registry, Codewind, Appsody
+duration: 5 minutes
+permalink: containerregistry
+type: document
+parent: installoncloud
+order: 1
+---
+# Specifying Container Registries for Codewind 
+
+When you run Codewind on Kubernetes, Codewind uses [`buildah`](https://github.com/containers/buildah) to build container images from Dockerfiles and then push the images to a container registry that you specify.
+
+There are two scenarios in Codewind where you need to specify what container registry to use:
+
+1. If you are running in Che and creating a Codewind style project, you need to specify zero or one container registry so Codewind can push images to that specific registry. 
```

Comment:
If they are creating codewind style projects, the definitely need to specify a registry to push to, so change `...you need to specify zero or one container registry...` to `...you need to specify an image registry...`

Reply:
It's for Che and for hybrid.